rpmconf/Install/Util/utilfunc.sh

```diff
@@ -1695,7 +1695,28 @@ removeErrorMessage() {
   exit 1
 }
 
+removeNalpeironContainer() {
+	isInstalled "zimbra-nalpeiron-offline-daemon"
```

Comment:
@umagmrit @RakeshAMore - do we need to hide the `nalpeiron` reference from the container name?

Reply:
No need, check my comment on other PR https://github.com/Zimbra/zm-core-utils/pull/149#discussion_r1534285243